backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -453,7 +432,7 @@ static int validateSingularESL(size_t *bytesRead, const unsigned char *esl,
 		return ESL_FAIL;
 	}
 	// get certificate
-	cert_size = get_esl_cert((const char *)esl, sigList,
+	cert_size = get_esl_cert((const char *)esl, eslvarsize,
```

Comment:
We might want to revisit `get_esl_cert` in skiboot sometime. I remember why I implemented it differently, I did not like how we call `get_esl_signature_list` inside this function despite already having aquired an ESL list object in all of the calling functions. Its a small function, that is more or less a type cast but it irked me.  For not though, it looks good to me.

Reply:
I'm certainly very happy for you to do that - if you want to propose a patch to skiboot we can rejig this patch to use the updated skiboot version. I will hold off on merging for now.

Reply:
I say just merge this. We can add this to the TODO, it's a very small fix that ultimately wouldn't change much. Another thing I would like to add to TODO is change `get_esl_signature_list` to something like `get_efi_signature_list` or `get_efi`, its like saying "ATM machine", the 'M' stands for machine! Anyhows, I am just showing how pedantic I am and that I think these little things can wait until we get bored.

Reply:
I may be wrong, but I think that particular function is more of an unpacking of the ESL than actually acquiring one. I think it is grounds for renaming (at some point, not a priority), though I will side channel you a potential concern with naming.